test/features/fixtures/test-app/run-bugsnag-expo-cli-install

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/expect -f
+
+set timeout -1
+
+# install
+spawn npx bugsnag-expo-cli install
+
+expect "@bugsnag/expo already appears to be installed, but is missing dependencies. Do you want to install them?"
+send "y\r"
+
+expect "If you want to install @bugsnag/expo *"
+send "\r"
+
```

Comment:
this is where we should expect to see "Do you want to automatically upload source maps to Bugsnag? (this will modify your app.json)"